Doc/library/array.rst

```diff
@@ -266,4 +266,3 @@ Examples::
 
    `NumPy <https://numpy.org/>`_
       The NumPy package defines another array type.
-
```

Comment:
Removing the extra newline is correct, but extraneous to the issue.  I am leaving it, but normally better not to toss such things in.